packages/amplify-provider-awscloudformation/src/__tests__/graphql-resource-manager/diff-test-helper.ts

```diff
@@ -1,35 +1,54 @@
-import * as gsiTestHelper from './gsi-test-helpers';
 import { diff as getDiffs } from 'deep-diff';
 import { DynamoDB } from 'cloudform';
+import { $TSAny } from 'amplify-cli-core';
+import * as gsiTestHelper from './gsi-test-helpers';
 import { DiffableProject } from '../../graphql-resource-manager/utils';
 
+/**
+ * Get a test Diffable project
+ */
 export const getDiffedProject = (
```

Comment:
AFAICT we don't actually have any tests on this, right?

Reply:
Yes, that's right. The module `GraphQLResourceManager` is not unit testable. This needs to be refactored little bit to be unit testable. However this are e2e tests to cover this code path.
https://github.com/aws-amplify/amplify-cli/blob/master/packages/amplify-e2e-tests/src/__tests__/schema-iterative-update-1.test.ts
https://github.com/aws-amplify/amplify-cli/blob/master/packages/amplify-e2e-tests/src/__tests__/schema-iterative-update-2.test.ts
https://github.com/aws-amplify/amplify-cli/blob/master/packages/amplify-e2e-tests/src/__tests__/schema-iterative-update-3.test.ts
https://github.com/aws-amplify/amplify-cli/blob/master/packages/amplify-e2e-tests/src/__tests__/schema-iterative-update-4.test.ts